### PR TITLE
Remove CloudBees branding

### PR DESCRIPTION
**Pull request by jglick**
Was always inappropriate, and especially given that it is up for adoption.

@reviewbybees esp. @HRMPW; @Tyler-Yates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 This plugin allows to define build environment as a Dockerfile, stored in project SCM.
-see also http://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Docker+Custom+Build+Environment+Plugin
+see also http://wiki.jenkins-ci.org/display/JENKINS/Docker+Custom+Build+Environment+Plugin
```
Comment (jglick) [inline README.md]: Should not be merged unless and until the new page is created and old deprecated with a redirect.

**Comment by HRMPW** [comment]
👍  yes, this was always individually maintained in the past.

**Comment by jglick** [comment]
@reviewbybees done

File: pom.xml [hunk 2]
```diff
@@ -8,26 +8,17 @@
...
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Docker+Custom+Build+Environment+Plugin</url>
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/Docker+Custom+Build+Environment+Plugin</url>
 
   <licenses>
     <license>
       <name>MIT</name>
     </license>
   </licenses>
 
-  <developers>
```
Comment (oleg-nenashev) [inline pom.xml]: Would be great to have a `retired` flag just to give credit to original maintainers, but it is not how it works now

**Comment by oleg-nenashev** [comment]
Resolved the merge conflict

**Pull request merged**